rpmlint/configspec.cfg

```diff
@@ -42,6 +42,8 @@ ReleaseExtension = string(default='')
 RpmGamesGroup = string(default='Games')
 # Doc files to which end of line and UTF-8 checks should not be applied
 SkipDocsRegexp = string(default='\.(?:rtf|x?html?|svg|ml[ily]?)$')
+# Enable spell checking
+SpellCheck = boolean(default=True)
 # Whether to use the Enchant spell checker for spell checking
 UseEnchant = boolean(default=True)
```

Comment:
isn't that redundant now? I would prefer to keep the existing setting rather than adding a new one that has the same behavior..